smpp/smpptest/server.go

```diff
@@ -53,32 +53,42 @@ func nextMessageId() string {
 
 // NewServer creates and initializes a new Server. Callers are supposed
 // to call Close on that server later.
-func NewServer() *Server {
-	s := NewUnstartedServer()
+func NewServer(user, password string, port int) *Server {
```

Comment:
If we're going to take a port number as argument, better to allow passing ip:port like all other listeners. Yet, I find it wrong to have to do this for the test server.